webpack.ui.config.js

```diff
@@ -79,6 +79,10 @@ module.exports = [
       app: path.join(__dirname, 'src', 'app.tsx'),
       serviceWorker: path.join(__dirname, 'src', 'serviceWorkers', 'index.ts'),
     },
+    ignoreWarnings: [
```

Comment:
nit:
Maybe a bit cleaner?
I found out the other day how to fix this instead of ignoring it: with [webpack ContextReplacementPlugin](https://webpack.js.org/plugins/context-replacement-plugin/)

https://github.com/zkitter/ui/commit/4a99b3edf358a1702836554c7acb58db0b41eee5